crates/arroyo-connectors/src/iggy/iggy.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If possible, this should be converted to black and white to match with the other connector images. (If that's not too much trouble, I can just up a monochrome version).